source/includes/_tipser-elements.md

```diff
@@ -102,6 +102,77 @@ All properties are optional:
 property name | description |type | default value
 --------------|-------------|-----|---------------
 defaultAddedToCartPopup | controls whether default Added To Cart Popup is displayed | boolean | true 
+eventsHandlers | gives possibility to add handlers to Tipser Elements events | object | empty
+useDefaultErrorHandler | if set to false there will be no error message displayed in case of error | boolean | true
```

Comment:
> if set to false there will be no error message displayed in case of error

"error" seems to be duplicated. What do you think about making it simpler ?
"if false, default message won't be displayed if error happens"

Reply:
+1

Reply:
or if false and error happens, default message won't be displayed

Reply:
nice